tests/test_support.py

```diff
@@ -0,0 +1,52 @@
+"""
+test_support.py
+
+Tests support functions.
+"""
+from x12.support import is_x12_data, is_x12_file, os
+import pytest
+
+
+@pytest.mark.parametrize(
```

Comment:
This may be a bit too clever but it does help reduce copying/pasting boiler plate test case code.

Reply:
actually, that's pretty cool.  anything to reduce lines of code!